R/satterthwaite.R

```diff
@@ -89,6 +89,7 @@ NULL
 #'
 #' @keywords internal
 h_quad_form_vec <- function(vec, center) {
+  vec <- as.vector(vec)
```

Comment:
added because there are error in unit tests of emmeans(where vec is not a vector, but a matrix $ p \times 1$